docs/documentation/backends/cairomakie.md

```diff
@@ -55,3 +55,10 @@ Assuming that you have a `Plot` object `plt`, you can set `plt.rasterize = true`
 For example, if your Scene's resolution is `(800, 600)`, by setting `scale=2`, the rasterized image will have a resolution of `(1600, 1200)`.
 
 You can deactivate this rasterization by setting `plt.rasterize = false`.
+
+Example: 
+\begin{examplefigure}{format=svg}
+fig = Figure()
+v = rand(10000,2)
+scatter(fig[1,1], v[:,1], v[:,2], rasterize = true, markersize = 1.0)
```

Comment:
```suggestion
v = rand(10000,2)
scatter(v[:,1], v[:,2], rasterize = true, markersize = 1.0)
```

It looks like `scatter` does not return a save-able value, so hopefully this helps.